pkgs/test/lib/src/util/math.dart

```diff
@@ -2,18 +2,19 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'dart:convert';
 import 'dart:math' as math;
-import 'dart:typed_data';
 
-/// Returns a random base64 string containing [bytes] bytes of data.
-///
-/// [seed] is passed to [math.Random].
-String randomBase64(int bytes, {int? seed}) {
-  var random = math.Random(seed);
-  var data = Uint8List(bytes);
-  for (var i = 0; i < bytes; i++) {
-    data[i] = random.nextInt(256);
+final _rand = math.Random();
+
+/// Returns a random alphanumeric string ([a-zA-Z0-9]), which is suitable as
+/// a url secret.
+String randomUrlSecret(int length) {
```

Comment:
How about hardcoding the length, since it's a special purpose utility now? Is 24 the right number of characters to choose? This has a lot less entropy than it used to.

I think we could safely go up to 32 characters since that's how long the base64 encoded version would be. It's still a smaller secret size, but not enough to matter.

Reply:
sure, done